DSharpPlus/Enums/Interaction/DiscordSelectDefaultValueType.cs

```diff
@@ -0,0 +1,8 @@
+namespace DSharpPlus;
+
+public enum DiscordSelectDefaultValueType
```

Comment:
docs, and don't put this in Enums - DSharpPlus/Entities/Interaction/Components sounds like a fitting folder path